src/components-examples/material/form-field/form-field-prefix-suffix/form-field-prefix-suffix-example.ts

```diff
@@ -14,4 +14,8 @@ import {MatFormFieldModule} from '@angular/material/form-field';
 })
 export class FormFieldPrefixSuffixExample {
   hide = true;
+  clickEvent(event: any) {
```

Comment:
You can replace the any here with MouseEvent.

Reply:
Thanks for your valuable input. Will do the same 🙂

Reply:
> You can replace the any here with MouseEvent.

pushed the changes. please verify